database/migrations/2025_06_12_183542_remove_name_from_individuals_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
```

Comment:
I've been removing these docblocks, along the lines of: https://github.com/spatie/laravel-stubs/blob/main/stubs/migration.stub